backend/tournaments/tools.py

```diff
@@ -198,6 +198,62 @@ def get_tournaments(only_active=False):
   return res
 
 
+def get_pairings(tournament, round):
+  """Returns pairings for a given round.
+
+  Arguments:
+   tournament -- tournament id
+   round -- index of a round (1-based)
+
+  Return:
+   Dictionary: {
+     'tournament_id': <id>
+     'tournament_name': <long tournament name>
+     'round': round index,
+     'buckets': [{
+       'bucket': <bucket name>,
+       'games': [{
+          'id': <game id>
+          'white': <white player username, or None if round is bye>,
+          'black': <black player username, or None if round is bye>,
+          'white_country': <white player country code, or None>
```

Comment:
I suggest to return "--" in case of no country. "--" is the name of the unknown flag.